src/test/java/com/wikia/webdriver/testcases/adstests/TestUap.java

```diff
@@ -19,6 +19,9 @@
 public class TestUap extends TemplateNoFirstLoad {
 
   private static Dimension DESKTOP_SIZE = new Dimension(1920, 1080);
+  private final static String MOBILE_IN_CONTENT_CONTAINER = ".mobile-in-content";
+  private final static String MOBILE_PREFOOTER_CONTAINER = ".mobile-prefooter";
+  private final static String MOBILE_BOTTOM_LEADERBOARD_CONTAINER = ".mobile-bottom-leaderboard";
```

Comment:
![MINOR](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/severity-minor.png) Reorder the modifiers to comply with the Java Language Specification. [![rule](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/rule.png)](http://sonar-s1/coding_rules#rule_key=squid%3AModifiersOrderCheck)